src/jdk.compiler/share/classes/com/sun/tools/javac/main/JavaCompiler.java

```diff
@@ -977,6 +977,14 @@ public void compile(Collection<JavaFileObject> sourceFileObjects,
                 log.printVerbose("total", Long.toString(elapsed_msec));
             }
 
+            // In case an Abort was thrown before processAnnotations could be called,
+            // we could have deferred diagnostics that haven't been reported. At the
+            // time of this comment's writing, this can only ever occur through a
```

Comment:
looks good. Suggestion: I would remove the last sentence: `At the time of this comment...` documents an erroneous situation. We usually don't add that type of comments.

Reply:
Done